### ISSUE TITLE
[Bug]: `dist` not in 5.0.0 distributed code

### ISSUE THREAD

**Issue by imsanchez** [sonofmagic/tailwindcss-mangle#114]
### version

5.0.0

### Link to minimal reproduction

https://www.npmjs.com/package/tailwindcss-patch/v/5.0.0?activeTab=code

### Steps to reproduce

Hey! Looks like 5.0.0 missed the tsup build (`dist` folder): https://www.npmjs.com/package/tailwindcss-patch/v/5.0.0?activeTab=code

<img width="280" alt="Screenshot 2024-10-28 at 12 51 26 AM" src="https://github.com/user-attachments/assets/72af18d7-c007-4a35-9c89-07319070ae49">

### What is expected?

<img width="254" alt="Screenshot 2024-10-28 at 12 51 18 AM" src="https://github.com/user-attachments/assets/ba698664-cda4-4597-908e-6c10f27a543c">

### What is actually happening?

```brand:build: > brand@ build /Users/isaiah/Desktop/Esaias/codetrackr/apps/brand
brand:build: node:internal/modules/esm/resolve:264
brand:build:     throw new ERR_MODULE_NOT_FOUND(
brand:build:           ^
brand:build: 
brand:build: Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'node_modules/tailwindcss-patch/dist/cli.js' imported from node_modules/tailwindcss-patch/bin/tw-patch.js
brand:build:     at finalizeResolution (node:internal/modules/esm/resolve:264:11)
brand:build:     at moduleResolve (node:internal/modules/esm/resolve:924:10)
brand:build:     at defaultResolve (node:internal/modules/esm/resolve:1148:11)
brand:build:     at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:390:12)
brand:build:     at ModuleLoader.resolve (node:internal/modules/esm/loader:359:25)
brand:build:     at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:234:38)
brand:build:     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
brand:build:     at link (node:internal/modules/esm/module_job:86:36) {
brand:build:   code: 'ERR_MODULE_NOT_FOUND',
brand:build: }
brand:build: 
brand:build: Node.js v21.7.3
brand:build:  ELIFECYCLE  Command failed with exit code 1.
brand:build: ERROR: command finished with error: command (/Users/isaiah/Desktop/Esaias/codetrackr/apps/brand) /Users/isaiah/.nvm/versions/node/v21.7.3/bin/pnpm run build exited (1)
brand#build: command (/Users/isaiah/Desktop/Esaias/codetrackr/apps/brand) /Users/isaiah/.nvm/versions/node/v21.7.3/bin/pnpm run build exited (1)
```

**Comment by sonofmagic** [comment]
Yes, this was an oversight on my part during the packaging process. I will fix this issue as soon as possible.

**Comment by sonofmagic** [comment]
Fixed in `5.0.1` : 
![QQ_1730125808899](https://github.com/user-attachments/assets/c01fe93f-7bee-4908-894b-f96ae2f08379)
